lib/ledger_sync/ledgers/quickbooks_online/client.rb

```diff
@@ -4,9 +4,9 @@
 require_relative 'dashboard_url_helper'
 
 module LedgerSync
-  module Adaptors
+  module Ledgers
     module QuickBooksOnline
-      class Adaptor < LedgerSync::Adaptors::Adaptor
+      class Client < Ledgers::Client
```

Comment:
Does this still work without the LedgerSync module reference?

Reply:
Up!